go/cmd/vtctldclient/command/workflows.go

```diff
@@ -35,11 +40,49 @@ var (
 		Args:                  cobra.ExactArgs(1),
 		RunE:                  commandGetWorkflows,
 	}
+
+	getWorkflowsOptions = struct {
+		ShowAll bool
+	}{}
 )
 
-var getWorkflowsOptions = struct {
-	ShowAll bool
-}{}
+var (
+	Workflow = &cobra.Command{
+		Use:                   "Workflow",
+		Short:                 "Interface with vreplication workflows (Reshard, MoveTables, etc) in the given keyspace",
```

Comment:
"Manage" instead of "interface" ? (maybe not "manage" specifically but something more specific than "interface" for sure)